README.md

```diff
@@ -1,35 +1,80 @@
-# perseo
+# Perseo Context-Aware CEP
 
 [![FIWARE Processing](https://nexus.lab.fiware.org/static/badges/chapters/processing.svg)](https://www.fiware.org/developers/catalogue/)
-[![License: APGL](https://img.shields.io/github/license/telefonicaid/perseo-fe.svg)](https://opensource.org/licenses/AGPL-3.0)
-<br/>
+[![License: AGPLv3](https://img.shields.io/github/license/telefonicaid/perseo-fe.svg)](./LICENSE)
+[![Docker Status](https://img.shields.io/docker/pulls/fiware/perseo.svg)](https://hub.docker.com/r/fiware/perseo/)
+[![Support badge](https://img.shields.io/badge/tag-fiware--perseo-orange.svg?logo=stackoverflow)](https://stackoverflow.com/questions/tagged/fiware-perseo)
+<br>
+[![Documentation badge](https://img.shields.io/readthedocs/perseo.svg)](https://perseo.readthedocs.io/en/latest/)
 [![CI](https://github.com/telefonicaid/perseo-fe/workflows/CI/badge.svg)](https://github.com/telefonicaid/perseo-fe/actions?query=workflow%3ACI)
 [![Coverage Status](https://coveralls.io/repos/github/telefonicaid/perseo-fe/badge.svg?branch=master)](https://coveralls.io/github/telefonicaid/perseo-fe?branch=master)
 ![Status](https://nexus.lab.fiware.org/static/badges/statuses/perseo.svg)
 
-CEP implementation for IoT platform
+## Overview
 
-## Index
+Perseo is a Complex Event Processing (CEP) software designed to be fully _NGSIv2_-compliant. It uses NGSIv2 as the
+communication protocol for events, and thus, Perseo is able to seamless and jointly work with _context brokers_. The context broker 
+tested with Perseo and officially supported is [Orion Context Broker](https://github.com/telefonicaid/fiware-orion).
 
--   [Architecture](documentation/architecture.md)
--   [Logs & Alarms](documentation/logs.md)
--   O&M
+This project is part of [FIWARE](https://www.fiware.org). You can find more FIWARE components in the
+[FIWARE catalogue](https://catalogue.fiware.org).
 
-    -   [Deployment](documentation/deployment.md)
-    -   [Configuration](documentation/configuration.md)
-    -   [Administration](documentation/admin.md)
-    -   [Metrics API](documentation/metrics_api.md)
+| :books: [Documentation](https://perseo.readthedocs.io/en/latest) | :whale: [Docker Hub](https://hub.docker.com/r/fiware/perseo/) | :dart: [Roadmap](documentation/roadmap.md) |
```

Comment:
As discussed at skype, this should point to docs/README.md, and docs/README.md should be added with an index (similar to the old one) with all the different topics covered by documentation.

Reply:
Added a ToC under /docs/README.md and also a reference was placed on main README.md file:

https://github.com/telefonicaid/perseo-fe/blob/db61072bd52a6f2a1294bf761b21fb5f6d22b7f6/README.md#L78

Reply:
Nice! :)